android/sdk/src/test/java/com/taobao/weex/ui/component/WXTextTest.java

```diff
@@ -27,7 +32,9 @@
 import com.taobao.weex.dom.WXTextDomObject;
 import com.taobao.weex.dom.flex.Spacing;
 import com.taobao.weex.ui.SimpleComponentHolder;
-
+import com.taobao.weex.ui.flat.FlatGUIIContext;
```

Comment:
Is FlatGUIIContext write more character 'I'

Reply:
My mistake, thank you for figuring out.